### PR TITLE
Amls 5389 | New summary page layout - trading premises

### PR DESCRIPTION
**Pull request by slavvco**
As above.

## Related / Dependant PRs?

- none

## How Has This Been Tested?

- manually

## Checklist

- [ ] Breaking change (fix or feature that would cause existing functionality to change).
- [x] Build passing.
- [ ] Unit tests have full coverage.
- [ ] Unit test approach and implementation has been reviewed with QA (testers).
- [ ] Requires acceptance test run.
- [x] Appropriate labels added.
- [x] RELEASE NOTES ADDED.

### CODE REVIEW

File: app/views/tradingpremises/summary_details.scala.html [hunk 1]
```diff
@@ -21,7 +21,7 @@
 @import models.businessmatching.MoneyServiceBusiness
 @import utils._
 
-@(model: TradingPremises, isMsb: Boolean, index: Int, hasOneService: Boolean, hasOneMsbService: Boolean = None)(implicit request: Request[_], m: Messages, lang: Lang)
```
Comment (ajchilts) [inline app/views/tradingpremises/summary_details.scala.html]: Is there a reason we have removed the optional param?

File: app/views/tradingpremises/summary_details.scala.html [hunk 2]
```diff
@@ -30,11 +30,13 @@
     title = Messages("summary.checkyouranswers.title") + " - " + Messages("summary.tradingpremises"),
     heading = header
 ) {
+    <dl class="cya-summary-list">
+
```
Comment (ajchilts) [inline app/views/tradingpremises/summary_details.scala.html]: Indentation?

File: app/views/tradingpremises/summary_details.scala.html (hunk 1 above)
Comment (slavvco) [reply app/views/tradingpremises/summary_details.scala.html]: Yes... there was None assigned to Boolean haha, but anyway this param is defaulted in controllers, so it's not necessary here.

File: app/views/tradingpremises/summary_details.scala.html (hunk 2 above)
Comment (slavvco) [reply app/views/tradingpremises/summary_details.scala.html]: Changed, is that what you wanted?

**Pull request merged**